ui/src/app/onboarding/page.tsx

```diff
@@ -0,0 +1,106 @@
+"use client"
+
+import { useState, ChangeEvent } from "react"
+import { postFirstUser } from "../queries"
+import { useMutation } from "react-query"
+import { useRouter } from "next/navigation"
+import { passwordIsValid } from "../utils"
+import { useAuth } from "../auth/authContext"
+
+
+export default function Onboarding() {
+    const router = useRouter()
+    const auth = useAuth()
+    const mutation = useMutation(postFirstUser, {
+        onSuccess: () => {
+            setErrorText("")
+            auth.setFirstUserCreated(true)
+            router.push("/login")
+        },
+        onError: (e: Error) => {
+            setErrorText(e.message)
+        }
+    })
+    const [username, setUsername] = useState<string>("")
+    const [password1, setPassword1] = useState<string>("")
+    const [password2, setPassword2] = useState<string>("")
+    const passwordsMatch = password1 === password2
+    const [errorText, setErrorText] = useState<string>("")
+    const [showPassword1, setShowPassword1] = useState<boolean>(false)
+    const handleUsernameChange = (event: ChangeEvent<HTMLInputElement>) => { setUsername(event.target.value) }
+    const handlePassword1Change = (event: ChangeEvent<HTMLInputElement>) => { setPassword1(event.target.value) }
+    const handlePassword2Change = (event: ChangeEvent<HTMLInputElement>) => { setPassword2(event.target.value) }
+    return (
+        <div style={{
+            display: "flex",
+            alignContent: "center",
+            justifyContent: "center",
+            flexWrap: "wrap",
+            height: "100vh",
+        }}>
+            <div className="p-panel" style={{
+                width: "45rem",
+                minWidth: "min-content",
+                minHeight: "min-content",
+            }}>
+                <div className="p-panel__header">
+                    <h1 className="p-panel__title">Welcome to GoCert</h1>
```

Comment:
```suggestion
                    <h1 className="p-panel__title">Initialize</h1>
```

Reply:
Plus I think everything should be in the box, the UI feels strange with content inside and outside of the box.